packages/patternfly-3/patternfly-react-extensions/less/classification-banner.less

```diff
@@ -0,0 +1,123 @@
+@classification-banner-height: 28px;
+@classification-banner-height-top: 3px;
+@classification-banner-padding-large: 42px;
+@classification-banner-padding-sml: 20px;
+
+.classification-banner-pf-banner{
+    background: @color-pf-black;
+    background-clip: padding-box;
+    color: white;
+    height: @classification-banner-height;
+    line-height: @classification-banner-height;
+    font-size: 10px;
+    position: relative;
+}
+
+.classification-banner-pf-banner-top{
+    width: 100%;
+    height: 31px;
+}
+.classification-banner-pf-banner-top::before{
+    content: '';
+    background: rgba(255,255,255,0.5);
+    display: block;
+    width: 100%;
+    height: 3px;
+}
+
+.classification-banner-pf-banner-bottom{
+    width: 100%;
+}
+
+.classification-banner-pf-hide{
+    display: none;
+}
+
+.classification-banner-pf-banner-classified{
+    background: @color-pf-green;
+}
+
+.classification-banner-pf-banner-unclassified{
+    background: @color-pf-red;
+}
+
+.classification-banner-pf-banner-proprietary-level{
+    background: @color-pf-blue;
+}
```

Comment:
@zeyap we should define color backgrounds as they are configurable. Color values need to be provided. Because of this, I don't think that we should have `classification-banner-pf-banner-classified`, `classification-banner-pf-banner-unclassified`, and `classification-banner-pf-banner-proprietary-level`.